packages/components/table-v2/src/use-table.ts

```diff
@@ -139,7 +139,7 @@ function useTable(props: TableV2Props) {
 
     if (
       unref(lastRenderedRowIndex) >= 0 &&
-      _totalHeight === scrollTop + unref(mainTableHeight) - unref(headerHeight)
+      _totalHeight.toFixed(2) <= (scrollTop + unref(mainTableHeight) - unref(headerHeight)).toFixed(2)
```

Comment:
```suggestion
      Math.abs(_totalHeight - (scrollTop + unref(mainTableHeight) - unref(headerHeight))) < 0.001
```
Because of precision issues, it seems impossible to determine the size relationship, so it is more reasonable to give a buffered area.